.github/workflows/tests_linux_x86.yml

```diff
@@ -0,0 +1,168 @@
+      max-parallel: 1
+
+    steps:
+      - name: Cancel Previous Runs
+        uses: styfle/cancel-workflow-action@0.11.0
+        with:
+          access_token: ${{ github.token }}
+
+      - uses: actions/setup-python@v4
+        name: Install Python
+        with:
+          python-version: '3.8'
+
+      - name: Remove Ubuntu unattended upgrades
+        run: sudo apt-get remove unattended-upgrades
+
+      - name: Install required packages
+        run: |
+          sudo apt-get update && sudo apt-get -y -q install gcc-${{ env.GCC_VERSION }} g++-${{ env.GCC_VERSION }} gcovr lcov
+          python -m pip install ninja cmake~=3.24.3 cuquantum
+
+      - name: Checkout pennyLane-lightning-gpu
+        uses: actions/checkout@v3
+
+      - name: Validate GPU version and installed compiler
+        run: |
+          nvidia-smi
+          /usr/local/cuda/bin/nvcc --version
+
+      - name: Build and run unit tests
+        run: |
+            cmake . -BBuild \
+              -DCMAKE_BUILD_TYPE=RelWithDebInfo \
+              -DPLLGPU_BUILD_TESTS=ON \
+              -DCMAKE_CXX_COMPILER="$(which g++-${{ env.GCC_VERSION }})" \
+              -DLIGHTNING_RELEASE_TAG="master" \
+              -DCMAKE_CUDA_COMPILER="/usr/local/cuda/bin/nvcc" \
+              -DCMAKE_CUDA_ARCHITECTURES="86" \
+              -G Ninja
+            cmake --build ./Build
+            cd ./Build
+            mkdir -p ./tests/results
+            ./pennylane_lightning_gpu/src/tests/runner_gpu --order lex --reporter junit --out ./tests/results/report_${{ github.job }}.xml
+
+      - name: Upload test results
+        uses: actions/upload-artifact@v3
+        if: always()
+        with:
+          name: ubuntu-tests-reports
+          path: ./Build/tests/results/report_${{ github.job }}.xml
+
+      - name: Build and run unit tests for code coverage
+        run: |
+            cmake . -BBuildCov \
+              -DCMAKE_BUILD_TYPE=Debug \
+              -DPLLGPU_BUILD_TESTS=ON \
+              -DPLLGPU_ENABLE_COVERAGE=ON \
+              -DCMAKE_CXX_COMPILER="$(which g++-${{ env.GCC_VERSION }})" \
+              -DLIGHTNING_RELEASE_TAG="master" \
+              -DCMAKE_CUDA_COMPILER="/usr/local/cuda/bin/nvcc" \
+              -DCMAKE_CUDA_ARCHITECTURES="86" \
+              -G Ninja
+            cmake --build ./BuildCov
+            cd ./BuildCov
+            ./pennylane_lightning_gpu/src/tests/runner_gpu
+            lcov --directory . -b ../pennylane_lightning_gpu/src --capture --output-file coverage.info
+            lcov --remove coverage.info '/usr/*' --output-file coverage.info
+            mv coverage.info coverage-${{ github.job }}.info
+
+      - name: Upload code coverage results
+        uses: actions/upload-artifact@v3
+        with:
+          name: ubuntu-codecov-results-cpp
+          path: ./BuildCov/coverage-${{ github.job }}.info
+
+      - name: Cleanup
+        if: always()
+        run: |
+          rm -rf ${{ steps.setup_venv.outputs.venv_name }}
```

Comment:
Do not need this line in specific since there are no python venvs being setup here

Reply:
Oops, yep, you are right.